pkg/virt-controller/services/template_test.go

```diff
@@ -3200,6 +3203,49 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.Containers[0].Resources.Requests.Memory().Value()).To(Equal(expectedMemory.Value()))
 			})
 		})
+
+		Context("with Virtual Machine name label in the template", func() {
```

Comment:
I think that using "template" in here is confusing and out of context.
Maybe "with Virtual Machine name label annotation" would be better.

Reply:
removed ` in the template`